LumbApp/Orquestador/Orquestador.cs

```diff
@@ -124,21 +125,60 @@ public void SetDatosDeSimulacion(Models.DatosPracticante datosPracticante, ModoS
 
 			tiempoTotalDeEjecucion = DateTime.UtcNow - tiempoInicialDeEjecucion;
 			Console.WriteLine("Tiempo total: "+tiempoTotalDeEjecucion);
-			//Guardar informe en archivo
-			//Informar a GUI con informe con un evento, que pase si el informe se genero bien, y si se guardó  bien (bool, bool) 
+			bool pdfGenerado = true; //Guardar informe en archivo
+			Informe informeFinal = CrearInformeFinal(informeZE, informeSI, pdfGenerado);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			IGUIController.MostrarResultados(informeFinal);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
+
+			//Informar a GUI con informe con un evento, que pase si el informe se genero bien, y si se guardó  bien (bool, bool)
```

Comment:
```suggestion
			Informe informeFinal = CrearInformeFinal(informeZE, informeSI, pdfGenerado);
			Console.WriteLine("Tiempo total: " + tiempoTotalDeEjecucion);
			IGUIController.MostrarResultados(informeFinal);
			//Informar a GUI con informe con un evento, que pase si el informe se genero bien, y si se guardó  bien (bool, bool)
```

No se si ese comentario es viejo o es un to-do valido, asi que por las dudas lo dejo